client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -37,7 +37,7 @@ class ThemesBanner extends PureComponent {
 	};
 
 	recordEvent = () => {
-		const { siteId, themeId, activeThemeId } = this.props;
+		const { siteId, themeId, activeThemeId, recordTracksEvent } = this.props;
```

Comment:
Unrelated nit: We try to alphabetize destructured vars. So since we're here, maybe we can change this line to 

```js
const { activeThemeId, recordTracksEvent, siteId, themeId } = this.props;
```

Reply:
Oh, good point! :D